js/client/modules/@arangodb/testsuites/rta_makedata.js

```diff
@@ -212,12 +212,14 @@ function makeDataWrapper (options) {
   if (localOptions.cluster && localOptions.dbServers < 3) {
     localOptions.dbServers = 3;
   }
+  localOptions.extraArgs['experimental-vector-index'] = true;
```

Comment:
Here we might still want to use the `--experimental-vector-index` flag, if this has to work with older versions. But if not, I would prefer `--vector-index`

Reply:
no this wrapper lives inside of the arangodb git repo and grows along with it.